packages/fast-tooling-react/app/pages/web-components.tsx

```diff
@@ -0,0 +1,213 @@
+import { Form, Viewer } from "../../src";
+import { FormProps } from "../../src/form/form.props";
+import { DesignSystemProvider } from "@microsoft/fast-jss-manager-react";
+import React from "react";
+import {
+    getDataFromSchema,
+    MessageSystem,
+    MessageSystemType,
+    SchemaDictionary,
+} from "@microsoft/fast-tooling";
+import FancyButton from "./web-components/fancy-button";
```

Comment:
What is FancyButton?

Reply:
"FancyButton" is actually a relatively common convention for informing someone that the component is an arbitrary abstraction. Since this is a testing folder I think this is fine. Below are some references to "Fancy" components:

https://reactjs.org/docs/forwarding-refs.html
https://developers.google.com/web/fundamentals/web-components/customelements#extend
https://www.webcomponents.org/element/SimpleElements/simple-button

Reply:
I think FancyButton is a fine name, none of these files are intended for export as they are all within the test app, I've added a `README.md` file to make that more clear to contributors.